cmake/resources/windows.rc.in

```diff
@@ -0,0 +1,34 @@
+#include <winver.h>
+
+IDI_ICON1   ICON    "@RES_DIR@/icon.ico"
+
+VS_VERSION_INFO VERSIONINFO
+    FILEVERSION        @PROJECT_VERSION_MAJOR@,@PROJECT_VERSION_MINOR@,@PROJECT_VERSION_PATCH@,@PROJECT_VERSION_TWEAK@
+    PRODUCTVERSION     @PROJECT_VERSION_MAJOR@,@PROJECT_VERSION_MINOR@,@PROJECT_VERSION_PATCH@,@PROJECT_VERSION_TWEAK@
+    FILEFLAGSMASK      VS_FFI_FILEFLAGSMASK
+    FILEFLAGS          VS_FF_SPECIALBUILD
+    FILEOS             VOS__WINDOWS32
+    FILETYPE           VFT_APP
+    FILESUBTYPE        0x0L
+BEGIN
+    BLOCK "StringFileInfo"
+    BEGIN
+        BLOCK "040904b0"
+        BEGIN
+            VALUE "ProductName",      "Chatterino"
+            VALUE "ProductVersion",   "@PROJECT_VERSION@"
+            VALUE "CompanyName",      "Chatterino, @PROJECT_HOMEPAGE_URL@"
+            VALUE "FileDescription",  "Chatterino @PROJECT_VERSION@ - @PROJECT_DESCRIPTION@"
```

Comment:
I would rather keep this short
I prefer `Chatterino VERSION` over `Chatterino VERSION - DESCRIPTION

Reply:
I prefer `Chatterino`, and on Nightly `Chatterino Nightly` to keep it simple

Reply:
a) `Chatterino`
b) `Chatterino VERSION`
c) `Chatterino NIGHTLY_TAG`
d) `Chatterino VERSION NIGHTLY_TAG`

Reply:
Nightly detection is currently done at runtime, so cant be added here unless major changes are made, so we are left with A or B

we have 3 votes for A going by upvotes, 
i would vote B

Reply:
I vote against having the full version there. There's no program I have opened that includes the full version. The only thing that comes close is having the year ($\equiv$ major version) of the program (e.g. Visual Studio).

Reply:
Let's go with A then

Reply:
pushed, final result: 
![img](https://github.com/Chatterino/chatterino2/assets/28986062/e67bb362-65f2-4e0c-9864-97d080e5fa06)